docs/csharp/whats-new/csharp-7.md

```diff
@@ -108,9 +108,8 @@ names to each of the members of the tuple:
 [!code-csharp[NamedTuple](../../../samples/snippets/csharp/new-in-7/program.cs#05_NamedTuple "Named tuple")]
 
 > [!NOTE]
-> The new tuples features require the `System.ValueTuple` type. For Visual Studio 15
-> Preview 5 and earlier preview releases, you must add the NuGet package "System.ValueTuple",
-> available in the pre-release stream.
+> The new tuples features require the `System.ValueTuple` type.
```

Comment:
This whole article is called "What's new in C# 7", so I think "new" will be appropriate here even in the future. Especially considering that there are "old" tuples (even though they don't actually have any language features supporting them).